github/event_types.go

```diff
@@ -752,6 +793,18 @@ type RepositoryVulnerabilityAlertEvent struct {
 	} `json:"alert,omitempty"`
 }
 
+// StarEvent is triggered when a star is added or removed from a repository.
+// The Webhook event name is "star".
+//
+// GitHub API docs: https://developer.github.com/v3/activity/events/types/#starevent
+type StarEvent struct {
+	// Action is the action that was performed. Possible values are: "created" or "deleted".
+	Action *string `json:"action,omitempty"`
+
+	// StarredAt is the time the star was created. Will be null for the deleted action.
```

Comment:
Let's please change this to:

```go
// StarredAt is the time the star was created. It will be null for the "deleted" action.
```

So that the auto-generated godocs will read better. Thanks.

Reply:
Done, thank you very much @gmlewis ! :)